test/e2e/storage/pd.go

```diff
@@ -429,8 +429,9 @@ var _ = framework.KubeDescribe("Pod Disks", func() {
 			podClient.Delete(host0Pod.Name, metav1.NewDeleteOptions(0))
 			detachAndDeletePDs(diskName, []types.NodeName{host0Name})
 			framework.WaitForNodeToBeReady(f.ClientSet, string(host0Name), nodeStatusTimeout)
+			framework.WaitForAllNodesSchedulable(f.ClientSet, nodeStatusTimeout)
```

Comment:
Yes it does, but "all nodes" only include nodes that are known to kubelet, which may not include the deleted node. The previous line checks for that case.